src/compiler.rs

```diff
@@ -4,7 +4,7 @@ use crate::modules::block::Block;
 use crate::translate::check_all_blocks;
 use crate::translate::module::TranslateModule;
 use crate::utils::{ParserMetadata, TranslateMetadata};
-use crate::{rules, Cli};
+use crate::rules;
```

Comment:
`Cli` is defined in `main.rs`, and would therefore be unavailable to a library crate, if we ever decide to separate the code into binary and library crates; removing references to it will help with that.